### ISSUE TITLE
M-Net loss too small

### ISSUE THREAD

**Issue by sanshibayuan** [tsing90/pytorch_semantic_human_matting#28]
Hi, when I do the pretrain_m_net stage, my losses are extrmely small, is that wrong? 
![image](https://user-images.githubusercontent.com/30117384/62762133-f7778780-baba-11e9-8dfe-c2e05e2b4e73.png)
